app/screens/create_direct_message/create_direct_message.tsx

```diff
@@ -1,84 +1,39 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useCallback, useEffect, useMemo, useReducer, useRef, useState} from 'react';
-import {useIntl} from 'react-intl';
-import {Keyboard, Platform, View} from 'react-native';
-import {SafeAreaView} from 'react-native-safe-area-context';
+import React, {useCallback, useState} from 'react';
+import {defineMessages, useIntl} from 'react-intl';
 
 import {makeDirectChannel, makeGroupChannel} from '@actions/remote/channel';
 import {fetchProfiles, fetchProfilesInTeam, searchProfiles} from '@actions/remote/user';
-import CompassIcon from '@components/compass_icon';
-import Loading from '@components/loading';
-import Search from '@components/search';
 import {General} from '@constants';
 import {useServerUrl} from '@context/server';
-import {useTheme} from '@context/theme';
-import {debounce} from '@helpers/api/general';
-import useNavButtonPressed from '@hooks/navigation_button_pressed';
 import {t} from '@i18n';
-import {dismissModal, setButtons} from '@screens/navigation';
+import MembersModal from '@screens/members_modal';
```

Comment:
The `MembersModal` is going to be used in the  `ManageMembers` and `Add People` screen that will be designed next.  That is the purpose of this refactor and creating this modal as its own component. The manage members will have different functions for getting the members and what to do when the action is taken (Add Member or Modify Members).

Reply:
But on both situations, it will be a component instantiated in a different screen, not a screen by itself, right? Then it should be under the `components` folder, not the `screens` folder.

Reply:
That's correct.  I moved the `members_modal` into a component and renameed to `UsersModal` mainly so the name change places it alphabetically next to the `user_*` components